ikarus/finiteelements/mechanics/enhancedassumedstrains.hh

```diff
@@ -40,6 +40,7 @@ public:
   using GridView                 = typename DisplacementBasedElement::GridView;
   using Traits                   = typename DisplacementBasedElement::Traits;
   using DisplacementBasedElement::localView;
+  using EASBaseType = std::conditional_t<Traits::mydim == 2, EAS::Q1E0<Geometry>, EAS::H1E0<Geometry>>;
```

Comment:
Maybe `NoEASType` furthermore this type should not be public since it is an impl details for isDisplacementBased